tests/test_convert_lit_checkpoint.py

```diff
@@ -0,0 +1,59 @@
+from unittest import mock
+from pathlib import Path
+from urllib.request import urlretrieve
+
+import pytest
+import torch
+
+wd = Path(__file__).parent.parent.absolute()
+
+
+def test_convert_lit_checkpoint(tmp_path):
+    from scripts.convert_lit_checkpoint import convert_lit_checkpoint
+
+    chkpt_name = "lit_model_finetuned"
```

Comment:
nit: usually we write ckpt

```suggestion
    ckpt_name = "lit_model_finetuned"
```